docs/tools/CLI/cli-test-debug.md

```diff
@@ -48,7 +48,7 @@ Allocated Stack: 32768 bytes
 Total Static RAM memory (data + bss): 6568 bytes
 Total RAM memory (data + bss + heap + stack): 104876 bytes
 Total Flash memory (text + data + misc): 48357 bytes
-Image: build\tests\K64F\GCC_ARM\TESTS\mbedmicro-rtos-mbed\mutex\TESTS-unit-myclass.bin
+Image: build\tests\K64F\GCC_ARM\TESTS\mbed_rtos\mutex\TESTS-unit-myclass.bin
```

Comment:
The actual path to `mbed-rtos` tests is `BUILD/tests/K64F/GCC_ARM/rtos/tests/TESTS/mbed_rtos/` after your PR in mbed-os.

But line is probably just a random path the original author used for demo only, and the value doesn't really matter.

Reply:
Yes, I updated with the correct path. Even though text is used for demo but still provide the right information would be better.